parm/ioda/bufr2ioda/bufr2ioda_mtiasi.yaml

```diff
@@ -0,0 +1,238 @@
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "{{ DMPDIR }}/{{ RUN }}.{{ PDY }}/{{ cyc }}/atmos/{{ RUN }}.t{{ cyc }}z.mtiasi.tm00.bufr_d"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          satelliteId:
+            query: "*/SAID"
+
+          sensorId:
+            query: "*/SIID[1]"
+
+          scanLineNumber:
+            query: "*/SLNM"
+
+          gqisFlagQual:
+            query: "*/QGFQ"
+
+          fieldOfViewNumber:
+            query: "*/FOVN"
+
+          sensorScanPosition:
+            sensorScanPosition:
+              fieldOfViewNumber: "*/FOVN"
+              sensor: iasi 
+
+          solarZenithAngle:
+            query: "*/SOZA"
+
+          solarAzimuthAngle:
+            query: "*/SOLAZI"
+
+          sensorZenithAngle:
+            query: "*/SAZA"
+
+          sensorAzimuthAngle:
+            query: "*/BEARAZ"
+
+          stationElevation:
+            query: "*/SELV"
+            type: float 
+
+          sensorViewAngle:
+            sensorScanAngle:
+              fieldOfViewNumber: "*/FOVN"
+              scanStart: -48.330
+              scanStep: 3.334
+              scanStepAdjust: 0.625 
+              sensor: iasi
+
+          fractionOfClearPixelsInFOV:
+            query: "*/IASIL1CS{1}/FCPH"
+#            type: float
+#            transforms:
+#               - scale: 0.01
+
+          sensorChannelNumber:
+            query: "*/IASICHN/CHNM"
+
+          # ObsValue
+          spectralRadiance:
+            spectralRadiance:
+              sensorChannelNumber: "*/IASICHN/CHNM"
+              startChannel: "*/IASIL1CB/STCH"
+              endChannel: "*/IASIL1CB/ENCH"
+              scaleFactor: "*/IASIL1CB/CHSF"
+              scaledSpectralRadiance: "*/IASICHN/SCRA"
+
+        splits:
+          satId:
+            category:
+              variable: satelliteId
+              map:
+                _3: metop-b
+                _4: metop-a
+                _5: metop-c
+
+    ioda:
+      backend: netcdf
+      obsdataout: "{{ COM_OBS }}/{{ RUN }}.t{{ cyc }}z.mtiasi_$(splitvar).tm00.nc" 
```

Comment:
any reason why this needs to be `mtiasi` and not `iasi`? I ask because our convention has been lately `$sensor_$platform` so it would match the CRTM coefficients. But if there are different input streams, then it makes sense to distinguish.

Reply:
@Cory For eacg IASI platform has three bufr sources:
gdas.t00z.mtiasi_metop-a.tm00.bufr_d --- normal feed
gdas.t00z.esiasi_metop-a.tm00.bufr_d --- EARS feed 
gdas.t00z.iasidb_metop-a.tm00.bufr_d--- direct broadcast feed

The one I am working on in this PR is the normal feed.  So, I use `mtiasi` in the filename to note that this file is converted from the normal feed.  
In the future, I will need to merge these three files into one, which will be called `iasi_metop-a` for the combined IODA file.